torch/_dynamo/variables/tensor.py

```diff
@@ -214,7 +214,6 @@ def create(cls, tx, proxy, example_value=None, **options):
             is_parameter = isinstance(example_value, torch.nn.Parameter)
             parameter_value = initial_example_value if is_parameter else None
 
-            example_value = _clone_input(example_value)
```

Comment:
Side note - it looks like the only use case we have for keeping real tensors around in dynamo today is in the export use case. There's an `output_graph.real_value_cache` that holds onto real tensor inputs, so we can run real compute during export.

I'd like to try to remove all clones of real tensors in dynamo, and I'm... pretty sure that in the export case, we want the tensors in the real_value_cache to be the originals, and not clones, that way mutations are properly reflected? I need to confirm that though. I'll probably put up a PR later and check CI / check with dynamo folks.